### ISSUE TITLE
"Citation" of text

### ISSUE THREAD

**Issue by jcowey** [hcayless/P3-processing#30]
https://github.com/hcayless/P3-processing/blob/main/articles/ClaytorSmithWarga_FourPoll_TaxReceiptsTrial_4/ClaytorSmithWarga_FourPoll_TaxReceiptsTrial_4.xml#L310-L311

splits a "citation" of a portion of two lines of Greek text into two paragraphs. That has knock on effects for HTML display.

needs to be revisited.

Maybe using citation mark up would be good. This would then also apply to the citation of languages other than Greek as well. Citation of other work of other scholars.

This would have to be signalled in the .docx mark up

**Comment by jcowey** [comment]
block quote 
has to be mark up

**Comment by jcowey** [comment]
[ClaytorSmithWarga_FourPoll_TaxReceiptsTrial_4new.docx](https://github.com/hcayless/P3-processing/files/6791363/ClaytorSmithWarga_FourPoll_TaxReceiptsTrial_4new.docx)

here is a version of the ClaytorSmithWarga article with new #quote mark up

**Comment by jcowey** [comment]
Citing verse patterns must be considered.

**Comment by jcowey** [comment]
use the mark up #blockQuote

is what we talked about 13.08.2021.

**Comment by jcowey** [comment]
This is also being discussed in:

https://gitlab.ub.uni-heidelberg.de/verlag/PapyrologicalPublicationPlatform/-/issues/24#note_27146

**Comment by lmaylein** [comment]
> This is also being discussed in:
> 
> https://gitlab.ub.uni-heidelberg.de/verlag/PapyrologicalPublicationPlatform/-/issues/24#note_27146

I guess I should give @hcayless  access to this gitlab repository.
@hcayless are you able to login to https://gitlab.ub.uni-heidelberg.de/cgi-bin/userid via Shibboleth?
If this is the case, please send me the displayed text by mail.

**Comment by hcayless** [comment]
Duke is in the list of providers, but I get the following error message:

```
We're sorry, but you cannot access this service at this time.

This service requires information about you that your identity provider (Duke University) did not release. To gain access to this service, your identity provider must release the required information.

Please visit the Duke University support page for further instructions.

You were trying to access the following URL:

For more information about this service, including what user information is required for access, please visit our information page.
```
The Duke support page is just the IT help page, and the GitLab about page is behind the login, so that's not very helpful...

**Comment by lmaylein** [comment]
Your IT department might not want to share attributes to our GitLab instance (actually only an automatically generated pseudonym would be necessary) because of a single user. Maybe it would be better to create a local account in our GitLab then?

**Comment by hcayless** [comment]
If you wrap quotes in:

```
#blockQuote
Text being quoted....
#endBlockQuote
```
It will now be turned into `<quote>` with `<lb/>`s for line breaks.

**Comment by jcowey** [comment]
Would it produce something like this: 

https://github.com/jcowey/P3/blob/master/pylon/pylon1zellmann-rohrer/zellmann_plond3_868.xml#L56-L71

**Comment by hcayless** [comment]
Yes, that's exactly the idea. We may still need to mess around a little with whitespace. See https://github.com/hcayless/P3-processing/blob/92ec236b2f1bceb178e5b99ec14b4069312ca5a7/articles/ClaytorSmithWarga_FourPoll_TaxReceiptsTrial_4/ClaytorSmithWarga_FourPoll_TaxReceiptsTrial_4.xml#L321-L325
